WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -13,8 +13,10 @@
 NSString * const ReaderTopicDidChangeViaUserInteractionNotification = @"ReaderTopicDidChangeViaUserInteractionNotification";
 NSString * const ReaderTopicDidChangeNotification = @"ReaderTopicDidChangeNotification";
 NSString * const ReaderTopicFreshlyPressedPathCommponent = @"freshly-pressed";
-static NSString * const ReaderTopicCurrentTopicURIKey = @"ReaderTopicCurrentTopicURIKey"; // Deprecated
-static NSString * const ReaderTopicCurrentTopicPathKey = @"ReaderTopicCurrentTopicPathKey";
+NSString * const ReaderTopicReadItLaterTitle = @"Read It Later";
```

Comment:
We'd probably want to localize the title via `NSLocalizedString`, perhaps inline vs defining a constant.